src/python/BuildingControlsSimulator/DataClients/WeatherChannel.py

```diff
@@ -49,6 +52,7 @@ class WeatherChannel(DataChannel):
 
     epw_backfill_columns = attr.ib()
 
+    #QUESTION could this be used for backfilling missing temp and rh?
```

Comment:
this gets into the data providence question, we know that we dont have radiation for the weather stations in flat weather (data from dark sky) - so backfilling from a different source doesnt raise issues because there is no other source. We could potentially backfill missing temp and rh data, but we may have issues if the two different weather stations diverge.

Reply:
I was also thinking we could completely rely on nsrdb for temp and rh.  not just a backfill.  it would be akin to having real world variance from forecast.  thoughts on that?

Reply:
we could add that as an optional feature, in general I think it works best if we use the data that the thermostats actually have access to/generate as the primary source.

Reply:
k let's park the idea for later.